### DISCUSSION TITLE
Different Wallpaper

### DISCUSSION THREAD

**Discussion by alex431** [linuxmint/discussions#471] [Ideas]
Is it possible to add built-in support for setting a different wallpaper per screen/monitor similar to xfce?

**Comment by huuxfluux** [comment]
THIS!
I came from KDE to Mint Cinnamon and i thought this was a obvious function.

**Comment by alex431** [comment]
Kde and cinnamon utlizes two different toolkits. Kde is built with qt toolkit, and Cinnamon is built with gtk toolkit. Cinnamon shares the same foundation as gnome stack technologies whereas kde is built from scratch.
________________________________

**Reply by huuxfluux** [reply]
Thx for the explanation. Still tho, this would be a good feature!
